examples/profile-client/src/test/java/com/palantir/example/profile/ProfileStoreTest.java

```diff
@@ -42,37 +42,30 @@
 import com.palantir.util.crypto.Sha256Hash;
 
 public class ProfileStoreTest {
-    TransactionManager txnMgr;
-    UserProfile user = UserProfile.newBuilder().setBirthEpochDay(0).setName("first last").build();
-    public static final byte[] IMAGE = new byte[] {0, 1, 2, 3};
+    private static final byte[] IMAGE = new byte[] {0, 1, 2, 3};
+    private static final UserProfile USER = UserProfile.newBuilder()
+            .setBirthEpochDay(0)
+            .setName("first last")
+            .build();
+
+    private final TransactionManager txnMgr = InMemoryAtlasDbFactory
+            .createInMemoryTransactionManager(ProfileSchema.INSTANCE);
```

Comment:
A new class is instantiated for each test run so this should not be a problem (with default junit runner).  I'm learning from @SerialVelocity that @Before is valuable because you can use it to perform set up actions that might result in Exceptions being thrown.